app/models/dependent.rb

```diff
@@ -101,8 +148,54 @@ def age_at_end_of_year(tax_year)
   end
 
   def eligible_for_child_tax_credit?(tax_year)
-    # add additional eligibility logic
-    age_at_end_of_year(tax_year) < 17
+     age_at_end_of_year(tax_year) < 17
+  end
+
+  def qualifying_child_relationship?
+    QUALIFYING_CHILD_RELATIONSHIPS.include? relationship
+  end
+
+  def qualifying_relative_relationship?
+    QUALIFYING_RELATIVE_RELATIONSHIPS.include? relationship
+  end
+
+  def qualifying_child?
+    qualifying_child_relationship? &&
+      meets_qc_age_condition? &&
+      meets_qc_misc_conditions? &&
+      meets_qc_residence_condition? &&
+      meets_qc_claimant_condition?
+  end
+
+  def meets_qc_age_condition?
+    (full_time_student_yes? && age_at_end_of_year(2020) < 24) || permanently_totally_disabled_yes? || age_at_end_of_year(2020) < 19
```

Comment:
does this make sense to be tax_year? it worries me that we're going to have to change this once we start filing 2021 returns.